packages/loading-indicator/package.json

```diff
@@ -25,15 +25,12 @@
   },
   "dependencies": {
     "@leafygreen-ui/emotion": "^4.0.4",
-    "@leafygreen-ui/lib": "^10.0.0",
+    "@leafygreen-ui/lib": "^10.3.3",
     "@leafygreen-ui/palette": "^4.0.4",
     "@leafygreen-ui/tokens": "^2.1.0",
     "@leafygreen-ui/typography": "^16.4.0",
     "@lottiefiles/react-lottie-player": "^3.5.3"
   },
-  "devDependencies": {
-    "jest-canvas-mock": "^2.5.1"
```

Comment:
do we no longer need this as a dev dep?

Reply:
No; this PR adds it as a dev dep on our main repo itself since we're moving it to be a global setup job